src/mod-keepout.cpp

```diff
@@ -94,7 +111,7 @@ class KeepOutPlayerScript : public PlayerScript
 
```

Comment:
Missed one on L91/108 in OnPlayerLogin(Player*).

Reply:
thanks, that config wasn't apart of the original cached values so I missed it. oops.